### PR TITLE
Gnome 3.24 bumps

### PR DESCRIPTION
**Pull request by ttambet**
Here is some basic bumps for final Gnome
Incomplete set of Gnome 3.24.0.

### CODE REVIEW

File: dev-libs/gjs/Manifest [hunk 1]
```diff
@@ -1 +1,3 @@
 DIST gjs-1.46.0.tar.xz 490812 SHA256 2283591fa70785443793e1d7db66071b36052d707075f229baeb468d8dd25ad4 SHA512 14af40668e95d269ac87556c75cf45680c41bc8246f04c46726d34bd80e9344045c0a510567f156c2c25c1f10dd474dc38bd50581d1b2e5abe0daa0cf570e2bc WHIRLPOOL 4aaf910c344d7df028a52e43b60b49e4fb0aefd92968e068d0effc4817b0f6b3d388c8287c6f7e3f85bc92c9e5605c03ff4743677ebc95dcd383d5a84d06f4fa
+DIST gjs-1.47.0.tar.xz 513480 SHA256 9255785ec704bda06d4a8300c9010efbdb83208f9496b0f3fe2799e872b411df SHA512 ace6efa3ac6c8b240d94ebadc0c2a42fe30dcb199d2d378cb6cd3f161e1fd10264de47c65fa9888f17ed60066138392b38a478526ed1b05ff683a456ef691a8c WHIRLPOOL 3948c017f222600847b749b23138a60bfa0cf5327fd2297f79d5a6b458b48cae0f4f5b35a22ecf090fa915c0a1d4e2a0378ed414885cbfdb85fde8dea5efcfa2
```
Comment (Moiman) [inline dev-libs/gjs/Manifest]: Why are you adding both 1.47.0 and 1.48.0?

File: app-text/gspell/Manifest [hunk 2]
```diff
@@ -1 +1,2 @@
 DIST gspell-1.2.3.tar.xz 395360 SHA256 a64e158e47a6e2b8ec9a3a1fea98a416f78a13e275c909b2ed488494ff835f82 SHA512 b45f17e8707187a070d4c5d4f91e4bcbd4108e596519c2900d7ca1f35050b6b397ed8c012fa25e179dddaf8b9c3a356b26c6070698f8333dff00c57a72e47198 WHIRLPOOL 1b6f22740d2b5597c20aa6d84e3e9e0b0b50c2fccfd811f49c99dff5e3762bc91a7582f223711f0f37a95c37d302b40697bd3517c170e8991cad4d556398875f
+DIST gspell-1.4.0.tar.xz 408852 SHA256 8d317c6bf6ae298b0a6c89aed63bc133c85c8ce7e8e799d870da560ad10b2d4e SHA512 f3b151e4785ea28535bc56ba9d28e7f10afc7ce03df111a52ee3b884f6e7e9730dd99cf0a1a26b8aa9bc825d1328868e269548939afcc65c06c9292ef2a824c7 WHIRLPOOL 94378578465b3e7672020667877f7031489878357a94e5936117bd25ec6418770d06bc1e4239869322e6441950b5a91658d17fd4952fdc62c45775e8c561e0eb
```
Comment (Moiman) [inline app-text/gspell/Manifest]: This is in wrong commit.

File: dev-libs/gjs/gjs-1.47.91.ebuild [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2017 Gentoo Foundation
```
Comment (Moiman) [inline dev-libs/gjs/gjs-1.47.91.ebuild]: This has misleading commit message: " x11-libs/vte: bump to 0.47.90"

**Comment by leio** [comment]
First 6 commits merged (with a commented out eapply from dev-libs/glib fixed up to remove as fixed upstream) together with a followup commit of phodav bump that I noticed as needed for new glib compatibility due to them having included the new g_uuid_* symbols and thus clashing as one of the functions was renamed during glib merge. Rest of it needs another day for review in my case. Feel free to rebase what's left, but I can manage with the current commits just fine as well, unless something to fix up is found by others meanwhile.

**Comment by leio** [comment]
Might be hard to believe, but all in tree now with follow-up fixes, sync with other in-tree changes and/or bumps

**Comment by lkraav** [comment]
Merry xmas @leio 👍 💯 🥇

**Pull request closed without merging**